Jenkinsfile

```diff
@@ -51,6 +52,15 @@ pipeline {
             }
         }
 
+        stage('Subsystem Test onprem') {
+            steps {
+                sh "make podman-pull-service-from-docker-daemon"
+                sh "export PODMAN_PULL_FLAG='--pull never'; make deploy-onprem-for-subsystem"
+                sh "export GINKGO_FOCUS_FLAG=-ginkgo.focus=minimal-set; make test-onprem"
```

Comment:
You could just call `make test-onprem FOCUS=minimal-set`
`GINKGO_FOCUS_FLAG` is a wrapper

Reply:
Done.